### DISCUSSION TITLE
Change wording from 'login with oauth' to 'login with OSM'

### DISCUSSION THREAD

**Discussion by thibaultmol** [Zverik/every_door#812] [Ideas]
you and I know what Oauth means, but the average person doesn't. so it probably makes sense to (maybe show an osm icon) and say the text 'login with osm' instead when you're logging in on the app

**Comment by fred-jonkhart** [comment]
I suggest 'login into osm' iso 'login with ...".  'with' denotes the mechanism of authentication. 'into' denotes the target.

**Comment by Zverik** [comment]
Good idea, thank you. I'll be reworking the entire accounts system to allow for signing it to different systems, so I'll keep this in mind.
